analysis/deep2_r23_o32/general.py

```diff
@@ -209,12 +215,32 @@ def run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=False,
                                     apply_dust=apply_dust, revised=revised,
                                     individual=individual, log=log)
 
-    '''
+    # This fits a three order polynomial without a lO32 component
+    curvefit_analysis.run_experiment_Zcal(fitspath, fitspath_curvefit,
+                                          fitspath_ini, secondorder=False,
+                                          threevariable=False, raw=raw,
+                                          apply_dust=apply_dust,
+                                          revised=revised,include_rlimit=True)
+
+    # This fits a second order polynomial with a lO32 component
+    curvefit_analysis.run_experiment_Zcal(fitspath, fitspath_curvefit,
+                                          fitspath_ini, secondorder=True,
+                                          threevariable=True, raw=raw,
+                                          apply_dust=apply_dust,
+                                          revised=revised, include_rlimit=True)
+
+    # This fits a second order polynomial without a lO32 component
+    curvefit_analysis.run_experiment_Zcal(fitspath, fitspath_curvefit,
+                                          fitspath_ini, secondorder=True,
+                                          threevariable=False, raw=raw,
+                                          apply_dust=apply_dust,
+                                          revised=revised, include_rlimit=True)
+
```

Comment:
Similar to other comment, you can create a `dict` of common inputs to pass in and have different parameter settings for `secondorder` and `threevariable`.

Reply:
applied